app/src/main/java/com/hedvig/app/feature/referrals/ReferralsReceiverActivity.kt

```diff
@@ -21,7 +24,9 @@ class ReferralsReceiverActivity : BaseActivity(R.layout.referrals_receiver_activ
         super.onCreate(savedInstanceState)
 
         binding.apply {
-            referralViewModel.redeemCodeStatus.observe(this@ReferralsReceiverActivity) { startChat() }
+            referralViewModel.viewState.onEach {
```

Comment:
Same comment as the other place where we're observing a flow instead of a LiveData